config/observability/README.md

```diff
@@ -0,0 +1,48 @@
+# Observability stack guide
+
+## Deploying the observabilty stack
+
+```bash
+./bin/kustomize build ./config/observability/| docker run --rm -i ryane/kfilt -i kind=CustomResourceDefinition | kubectl apply --server-side -f -
+./bin/kustomize build ./config/observability/| docker run --rm -i ryane/kfilt -x kind=CustomResourceDefinition | kubectl apply -f -
+```
+
+This will deploy prometheus, alertmanager and grafana into the `monitoring` namespace,
+along with metrics scrape configuration for Istio and Envoy.
+
+## Accessing Grafana & Prometheus
+
+Use port forwarding to access Grafana & Prometheus:
+
+```bash
+kubectl -n monitoring port-forward service/grafana 3000:3000
+```
+
+The Grafana UI can be found at [http://127.0.0.1:3000/](http://127.0.0.1:3000/).
```

Comment:
i think adding a note of the username and password for default grafana would be useful for people not because it took me ages to remember or anything like that 😵‍💫